terraform/variables/prod-intl.tfvars

```diff
@@ -57,9 +57,9 @@ ingestors = {
   }
 }
 cluster_settings = {
-  initial_node_count = 3
-  min_node_count     = 3
-  max_node_count     = 3
+  initial_node_count = 9
```

Comment:
I'm fine with this change, just calling it out in case it was not intended to be included in the PR.

Reply:
It was intended and necessary so that we won't shrink the cluster back to 3 nodes when we deploy this change.